vulkancapsviewer.cpp

```diff
@@ -80,9 +84,16 @@ OSInfo getOperatingSystem()
 {
     // QSysInfo works for all supported operating systems
     OSInfo osInfo = {};
+    // GhostBSD - QSysInfo::productType().toStdString() returs "unknown"
+#ifndef USE_UTSNAME
     osInfo.name = QSysInfo::productType().toStdString();
-    osInfo.architecture = QSysInfo::buildCpuArchitecture().toStdString();

```

Comment:
This line is missing in your PR. It's an important info required by the database.

Reply:
OK